grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -659,15 +661,35 @@ class ReportService implements ApplicationContextAware {
     }
 
     List getOnOrderSummary(Location location) {
+        String locale = LocalizationUtil.localizationService.getCurrentLocale().toLanguageTag()
+
         String query = """
             select 
                 product.product_code as productCode, 
                 product.name as productName, 
                 oos.quantity_ordered_not_shipped as qtyOrderedNotShipped,
                 oos.quantity_shipped_not_received as qtyShippedNotReceived, 
-                ps.quantity_on_hand as qtyOnHand
+                ps.quantity_on_hand as qtyOnHand,
+                product_group.name as productFamilyName,
+                category.name as productCategoryName,
+                (
+                select s.name from synonym s
+                where s.product_id = product.id
+                and s.synonym_type_code = '${SynonymTypeCode.DISPLAY_NAME}'
```

Comment:
Let's keep one convention and pass `SynonymTypeCode.DISPLAY_NAME` and `locale` as query params (same as `locationId`)